src/features/venueMap/components/VenueMapBlock/VenueMapBlock.tsx

```diff
@@ -4,19 +4,25 @@ import styled from 'styled-components/native'
 
 import { VENUE_MAP_BACKGROUND } from 'features/venueMap/components/VenueMapBlock/VenueMapBackground'
 import { useHandleFocus } from 'libs/hooks/useHandleFocus'
+import { Touchable } from 'ui/components/touchable/Touchable'
 import { InternalTouchableLink } from 'ui/components/touchableLink/InternalTouchableLink'
 import { getSpacing, Spacer, Typo } from 'ui/theme'
 import { customFocusOutline } from 'ui/theme/customFocusOutline/customFocusOutline'
 import { getHeadingAttrs } from 'ui/theme/typographyAttrs/getHeadingAttrs'
 
-export const VenueMapBlock: FunctionComponent = ({ ...props }) => {
+type Props = {
+  onPress?: VoidFunction
+}
+
+export const VenueMapBlock: FunctionComponent<Props> = ({ onPress, ...props }) => {
   const focusProps = useHandleFocus()
+  const TouchableContainer = onPress ? StyledTouchable : StyledInternalTouchableLink
```

Comment:
J'aurai fait la même chose pour les props aussi, pour ne pas passer des props non reconnus au composant
```suggestion
  const TouchableContainer = onPress ? StyledTouchable : StyledInternalTouchableLink
  const touchableProps = onPress ? { onPress } : { navigateTo: { screen : 'VenueMap' } }
```
Tu pourras passer` {...touchableProps}` ensuite à `TouchableContainer`